src/wp-admin/css/colors/modern/colors.scss

```diff
@@ -0,0 +1,17 @@
+$base-color: #1e1e1e;
+$highlight-color: #3858e9;
+$notification-color: #e26f56;
+$menu-submenu-focus-text: #33f078;
+
+@import "../_admin.scss";
+
+// Customize block editor colors.
+:root {
+	--wp-admin-theme-color: #3858e9;
+	--wp-admin-theme-color-darker-10: #1D35B4;
+	--wp-admin-theme-color-darker-20: #152A9C;
+}
```

Comment:
Right now, this is done on the Gutenberg side for all admin schemes, I think it makes sense to do this here but we should be consistent and do the same for all admin themes :).

This could be done using a mixin here and would allow us to remove the code from the Gutenberg plugin.

Another positive aspect of doing so is that we could start using the CSS variables in the whole Admin too.

Reply:
It looks like https://core.trac.wordpress.org/ticket/49999 is looking to refactor these schemes to use CSS variables entirely, that would help addresse these comments. In the mean time maybe I should remove the editor specific ones from here, and do the same for this theme, in the block editor, as we do for all the others, just in the name of shipping.

Reply:
works for me @jasmussen